doc/lightweight-styling.md

```diff
@@ -22,16 +22,31 @@ uid: uno.themes.lightweightstyling
 All interactive controls have multiple states, such as **PointerOver** (mouse is hovered over), **Pressed** (control is pressed on), and **Disabled** (control is not interactable). These states are appended onto the endings of the resource keys: ButtonForeground*PointerOver*, ButtonForeground*Pressed*, and ButtonForeground*Disabled*. Combined with these, the CheckBox and RadioButton controls also have **Checked** and **Unchecked** states. [These links](lightweight-styling#resource-keys) list the resource keys for each control.
 
 ```xml
-<Button Content="Overridden Button Style" Style="{StaticResource MaterialFilledButtonStyle}" BorderThickness="2">
+<Button Content="Default Button Style"
+		Style="{StaticResource FilledButtonStyle}" />
+
+<Button Content="Overridden Button Style"
+		Style="{StaticResource FilledButtonStyle}">
 	<Button.Resources>
-		<SolidColorBrush x:Key="FilledButtonForeground" Color="DarkGreen" />
-		<SolidColorBrush x:Key="FilledButtonBackground" Color="LightGreen" />
-		<SolidColorBrush x:Key="FilledButtonBorderBrush" Color="DarkGreen" />
+		<SolidColorBrush x:Key="FilledButtonForeground"
+							Color="DarkGreen" />
+		<SolidColorBrush x:Key="FilledButtonBackground"
+							Color="LightGreen" />
+		<SolidColorBrush x:Key="FilledButtonBorderBrush"
+							Color="DarkGreen" />
+	</Button.Resources>
+</Button>
 
-		<!-- Overriding the PointerOver brushes -->
-		<SolidColorBrush x:Key="FilledButtonForegroundPointerOver" Color="DarkRed" />
-		<SolidColorBrush x:Key="FilledButtonBackgroundPointerOver" Color="LightPink" />
-		<SolidColorBrush x:Key="FilledButtonBorderBrushPointerOver" Color="DarkRed" />
+<Button Content="Overridden Button Style (PointerOver)"
+		Style="{StaticResource FilledButtonStyle}">
+	<Button.Resources>
+		<Thickness x:Key="ButtonBorderThickness">2</Thickness>
+		<SolidColorBrush x:Key="FilledButtonForegroundPointerOver"
+							Color="DarkRed" />
+		<SolidColorBrush x:Key="FilledButtonBackgroundPointerOver"
+							Color="LightPink" />
+		<SolidColorBrush x:Key="FilledButtonBorderBrushPointerOver"
+							Color="DarkRed" />
 	</Button.Resources>
 </Button>
 ```
```

Comment:
Sorry I'm annoying :) but I formatted the XAML a bit better here so can you use this one:

```xml
<Button Content="Default Button Style"
		Style="{StaticResource FilledButtonStyle}" />

<Button Content="Overridden Button Style"
		Style="{StaticResource FilledButtonStyle}">
	<Button.Resources>
		<SolidColorBrush x:Key="FilledButtonForeground" Color="DarkGreen" />
		<SolidColorBrush x:Key="FilledButtonBackground" Color="LightGreen" />
		<SolidColorBrush x:Key="FilledButtonBorderBrush" Color="DarkGreen" />
	</Button.Resources>
</Button>

<Button Content="Overridden Button Style (PointerOver)"
		Style="{StaticResource FilledButtonStyle}">
	<Button.Resources>
		<Thickness x:Key="ButtonBorderThickness">2</Thickness>
		<SolidColorBrush x:Key="FilledButtonForegroundPointerOver" Color="DarkRed" />
		<SolidColorBrush x:Key="FilledButtonBackgroundPointerOver" Color="LightPink" />
		<SolidColorBrush x:Key="FilledButtonBorderBrushPointerOver" Color="DarkRed" />
	</Button.Resources>
</Button>
```

No need to change the screenshot, it looks good as is

Reply:
done